src/CloudNative.CloudEvents.Amqp/AmqpClientExtensions.cs

```diff
@@ -19,15 +19,15 @@ public static class AmqpClientExtensions
 
         public static bool IsCloudEvent(this Message message) =>
             HasCloudEventsContentType(message, out _) ||
-            message.ApplicationProperties.Map.ContainsKey(SpecVersionAmqpHeader);        
+            message.ApplicationProperties.Map.ContainsKey(SpecVersionAmqpHeader);
 
         public static CloudEvent ToCloudEvent(this Message message,
             CloudEventFormatter formatter,
             params CloudEventAttribute[] extensionAttributes)
         {
             if (HasCloudEventsContentType(message, out var contentType))
             {
-                return formatter.DecodeStructuredModeMessage(new MemoryStream((byte[])message.Body), new ContentType(contentType), extensionAttributes);
+                return formatter.DecodeStructuredModeMessage(new MemoryStream((byte[]) message.Body), new ContentType(contentType), extensionAttributes);
```

Comment:
Should structured mode do the same application data check?

Reply:
Yes, probably. The cast at the moment is somewhat unpleasant. Will leave that for a follow-up PR.